src/main/java/com/google/summit/translation/Translate.kt

```diff
@@ -1350,9 +1351,11 @@ class Translate(val file: String, private val tokens: TokenStream) : ApexParserB
       *visitSoslClauses(ctx.soslClauses()).bindings.toTypedArray(),
       ctx.boundExpression()?.let { visitBoundExpression(it) },
     ).filterNotNull()
-    return SoslExpression(toSourceString(ctx),
-                          bindings,
-                          toSourceLocation(ctx))
+    return SoslExpression(
+        toSourceString(ctx).trim().removeSurrounding("[","]"),
```

Comment:
this change looks unrelated, no?

Reply:
Errr, sorry, this is a bit from the other PR from earlier this week.

(P.S. That needs to be reapproved. It lost the approval bit after I made the requested changes.)

Reply:
Cleaned up.